app/scripts/components/dashboardComponents/treeList.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
remove PropTypes from React and just use `const {} = PropTypes`

Reply:
Mostly Notes for myself
Push up fixes for this